tests/signatures.rs

```diff
@@ -35,7 +35,7 @@ fn ed25519_key_and_ed25519_good_signature() {
     let ee = include_bytes!("signatures/ed25519.ee.der");
     let message = include_bytes!("signatures/message.bin");
     let signature = include_bytes!("signatures/ed25519_key_and_ed25519_good_signature.sig.bin");
-    assert_eq!(check_sig(ee, webpki::ED25519, message, signature), Ok(()));
+    assert_eq!(check_sig(ee, webpki::ring::ED25519, message, signature), Ok(()));
```

Comment:
Maybe we should be importing these algorithms at the top of the module, so that the full paths aren't repeated in every test, but just once at the top? That could be a separate commit.